### ISSUE TITLE
getting noflo-jekyll running

### ISSUE THREAD

**Issue by forresto** [noflo/noflo-jekyll#7]
I tried to get it running to build flowhub et al locally. First issue was that I had to run `npm install -g coffee-script`. Second issue seems deeper:

```
$ noflo-jekyll flowhub.io/ build/
NoFlo Jekyll /Users/f/src/flowhub.io -> /Users/f/src/build

/Users/f/bin/npm/lib/node_modules/noflo-jekyll/node_modules/noflo-routers/node_modules/nohoarder/lib/cache_storage.coffee:104
        throw new Error("No cache with key " + key + " to locate");
              ^
Error: No cache with key null to locate
  at _ref (/Users/f/bin/npm/lib/node_modules/noflo-jekyll/node_modules/noflo-routers/node_modules/nohoarder/lib/cache_storage.coffee:57:7)

...
```

**Comment by bergie** [comment]
This is caused by noflo/noflo-routers#4. Will be fixed by NoFlo 0.5

**Comment by bergie** [comment]
Should be fixed now in the git version. Will push to NPM as soon as Travis confirms
